examples/braket_features/Native_gates_on_IonQ.ipynb

```diff
@@ -0,0 +1,1666 @@
+{
```

Comment:
<p>If you want, you can make these (especially xx and yy) into single line functions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
true, I changed them to
```
def yy(qubit0, qubit1, theta):
    return IonQNativeCircuit().ms(qubit0, qubit1, pi / 2, pi / 2, theta)
```
now. which is pretty short and readable imo